Added batch normalized critic v1 and changed default in TD3

armenkasp opened the pull request:
Added batch normalization to critic to create critic-v1


Results from the unit test baselines and registry:

![image](https://github.com/JeffersonLab/SciOptControlToolkit/assets/155679987/ab9e0bc7-2b8d-42b5-b626-351379ed2c0f)

![image](https://github.com/JeffersonLab/SciOptControlToolkit/assets/155679987/f2ccb056-c179-4f7c-811a-00946f674232)

Kishanrajput reviewed (changes requested):
I do not see a purpose of having a separate version of critic to include one extra layer. This could be made configurable in v0 itself with an entry in cfg file that act as conditional. Am I missing something?

armenkasp commented:
> I do not see a purpose of having a separate version of critic to include one extra layer. This could be made configurable in v0 itself with an entry in cfg file that act as conditional. Am I missing something?

Removed the extra critic file and init call and moved the batch normalization into a conditional in the v0 critic with configuration control. Testing now.

armenkasp commented:
Configuration flag now controls if batch normalization is used in the critic

<img width="628" alt="Screenshot 2024-06-18 at 3 12 02 PM" src="https://github.com/JeffersonLab/SciOptControlToolkit/assets/155679987/1ed52082-574b-416d-8ba9-edeb533b2e00">
